packages/synapse-react-client/src/utils/functions/deepLinkingUtils.ts

```diff
@@ -93,24 +93,18 @@ export function updateUrlWithNewSearchParam(
 export function getQueryRequestFromLink(
   componentName: string,
   componentIndex: number,
-): QueryBundleRequest | undefined {
+): Partial<QueryBundleRequest> | undefined {
   const searchParamValue = getSearchParamValueFromUrl(
     componentName,
     componentIndex,
   )
 
-  let initQueryRequest: QueryBundleRequest | undefined = undefined
+  let initQueryRequest: Partial<QueryBundleRequest> | undefined = undefined
   if (searchParamValue) {
     const query = JSON.parse(searchParamValue) as Query
     if (query.sql) {
       initQueryRequest = {
         concreteType: 'org.sagebionetworks.repo.model.table.QueryBundleRequest',
-        partMask:
-          SynapseConstants.BUNDLE_MASK_QUERY_COLUMN_MODELS |
-          SynapseConstants.BUNDLE_MASK_QUERY_FACETS |
-          SynapseConstants.BUNDLE_MASK_QUERY_SELECT_COLUMNS |
-          SynapseConstants.BUNDLE_MASK_QUERY_RESULTS |
-          SynapseConstants.BUNDLE_MASK_QUERY_COUNT,
```

Comment:
This was probably a bug waiting to happen - when we populate the query from a search parameter, we shouldn't override the `partMask`. Now, this partial request gets merged in with the value passed from props. The `partMask` used will be specified by the component, which asks for exactly what it needs.